tests/Integration/Schema/Directives/HasManyDirectiveTest.php

```diff
@@ -158,6 +158,52 @@ public function testCanQueryHasManyPaginator(): void
         ])->assertJsonCount(2, 'data.user.tasks.data');
     }
 
+    public function testDoesNotRequireModelClassForPaginatedHasMany(): void
```

Comment:
I don't see the difference between this test and the previous one. Both doesn't have the modelClass set, just the type.

Reply:
The type name does not match the model name. It is not needed, the BC was that 4.4 forced them to line up.

Renamed to `type NotTheModelNameTask` for clarification.